### PR TITLE
[16.0] account_move_line_reconcile_manual: add write-off models

### PR DESCRIPTION
**Pull request by alexis-via**

### CODE REVIEW

File: account_move_line_reconcile_manual/security/ir.model.access.csv [hunk 1]
```diff
@@ -1,2 +1,4 @@
 id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
+access_account_reconcile_manual_model_full,Full access on account.reconcile.manual.model,model_account_reconcile_manual_model,account.group_account_manager,1,1,1,1
+access_account_reconcile_manual_model_read,Read access on account.reconcile.manual.model,model_account_reconcile_manual_model,account.group_account_manager,1,0,0,0
```
Comment (houzefa-abba) [inline account_move_line_reconcile_manual/security/ir.model.access.csv]: Copy-paste issue here, same group on both lines :P

File: account_move_line_reconcile_manual/models/account_reconcile_manual_model.py [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+
+class AccountReconcileManualModel(models.Model):
+    _name = "account.reconcile.manual.model"
+    _description = "Models for Manual Reconcile Write-off"
+    _check_company_auto = True
+    _rec_name = "ref"
+    _order = "sequence, id"
+
+    sequence = fields.Integer()
+    company_id = fields.Many2one("res.company", ondelete="cascade", required=True)
```
Comment (houzefa-abba) [inline account_move_line_reconcile_manual/models/account_reconcile_manual_model.py]: I would expect an SQL index on this field

**Review by houzefa-abba — CHANGES_REQUESTED** [review]
Great job thanks! 2 small things in this review

File: account_move_line_reconcile_manual/security/ir.model.access.csv (hunk 1 above)
Comment (alexis-via) [reply account_move_line_reconcile_manual/security/ir.model.access.csv]: Nice catch ! Fixed.

File: account_move_line_reconcile_manual/models/account_reconcile_manual_model.py (hunk 2 above)
Comment (alexis-via) [reply account_move_line_reconcile_manual/models/account_reconcile_manual_model.py]: done

**Review by houzefa-abba — APPROVED** [review]

**Pull request merged**